internal/tsc_wrapped/compiler_host.ts

```diff
@@ -13,6 +13,23 @@ export type ModuleResolver =
         ts.ResolvedModuleWithFailedLookupLocations;
 
 /**
+ * Narrows down the type of some properties from non-optional to required, so
+ * that we do not need to check presence before each access.
+ */
```

Comment:
This comment should be on the function, no?

Reply:
sure, I grabbed it from first-party, in the CL that syncs this I'll update that too